### DISCUSSION TITLE
Popover component backdrop color

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#42894] [Ideas]
Hi @zombieJ really would fine to add the backdrop layer for popover component, for a better UI.
Demo: https://tripfinder-hotel.vercel.app/listing

**Comment by Wxh16144** [comment]
in `5.6.0` version, token setting for components is supported separately, the following code may meet your needs

```tsx
import { ConfigProvider, Popover } from 'antd';
import React from 'react';

const content = (
  <div>
    <p>Content</p>
    <p>Content</p>
  </div>
);

const App: React.FC = () => (
  <ConfigProvider
    theme={{
      components: {
        Popover: {
          colorBgElevated: 'red' // here
        },
      },
    }}
  >
    <Popover content={content} title="Title">
      <button type="button">Hover me</button>
    </Popover>
  </ConfigProvider>
);

export default App;

```

**Reply by vitalijalbu** [reply]
@Wxh16144 no wrong, it applies color to the popover, but i need backdrop color.
https://codesandbox.io/s/basic-antd-5-6-3-forked-5dhs3t?file=/demo.tsx

**Reply by MadCcc** [reply]
What's the difference between `colorBgElevated` and what you want?

**Reply by vitalijalbu** [reply]
colorBgElevated applies color to the popover, but i want backdrop color. See codesandbox link pls.
